src/Core/Resolvers/CosmosMutationEngine.cs

```diff
@@ -49,6 +54,7 @@ private async Task<JObject> ExecuteAsync(IMiddlewareContext context, IDictionary
 
             if (string.IsNullOrEmpty(dataSourceName))
             {
+                // in rest scenario, dataSourceName could be empty.
```

Comment:
There should be no rest scenario for CosmosDb

Reply:
fixed this.